### PR TITLE
Modify postgres to make table and column names case sensitive

### PR DESCRIPTION
**Pull request by mkevinosullivan**
### WHY are these changes introduced?

Fixes Shopify/shopify-api-js#460

### WHAT is this pull request doing?

Adding `"` around table and column names so that case sensitivity is preserved.

## Type of change

- [x] Patch: Bug (non-breaking change which fixes an issue)
- [ ] Minor: New feature (non-breaking change which adds functionality)
- [ ] Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above
- [ ] I have added/updated tests for this change
- [ ] I have documented new APIs/updated the documentation for modified APIs (for public APIs)

### CODE REVIEW

**Comment by cquemin** [comment]
@mkevinosullivan I believe this will effectively "loose" all previously stored session since we will now be pointing at `ShopifySession` instead of `shopifysession` before your changes. If we do that should we then import all data ?

**Comment by mkevinosullivan** [comment]
> @mkevinosullivan I believe this will effectively "loose" all previously stored sessions since we will now be pointing at `ShopifySession` instead of `shopifysession` before your changes. If we do that should we then import all data ?

I'm going to add a flag to let the dev decide if they want case-sensitivity or not ... doing a migration here might be trickier ... although it's should just be as simple as running ...

```
ALTER TABLE shopifysession RENAME TO "ShopifySession";
ALTER TABLE "ShopifySession" RENAME COLUMN isonline TO "isOnline";
ALTER TABLE "ShopifySession" RENAME COLUMN onlineaccessinfo TO "onlineAccessInfo";
ALTER TABLE "ShopifySession" RENAME COLUMN accesstoken TO "accessToken";
```

Let me try writing a migration ...

**Comment by paulomarg** [comment]
If we have this sort of migration strategy in multiple offerings, should we bake the logic for it into the base package?

**Comment by cquemin** [comment]
@paulomarg we can but should we ? As I am starting to think that each provider will have its way to handle migration, maybe even a recommended migration manager .... It is a genuine question. As we are starting to be in muddy waters: borders between giving sample usages and  making tech choices.. Maybe we can simply keep it vanilla everywhere but this may mean rewriting a migration mechanism so it is future proof...

File: packages/shopify-app-session-storage-postgresql/src/__tests__/mytest.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import * as child_process from 'child_process';
```
Comment (cquemin) [inline packages/shopify-app-session-storage-postgresql/src/__tests__/mytest.ts:1]: Does this file need to stay?

File: packages/shopify-app-session-storage-postgresql/src/postgresql.ts [hunk 2]
```diff
@@ -176,4 +178,31 @@ export class PostgreSQLSessionStorage implements SessionStorage {
...
+
+    // Allow multiple apps to be on the same host with separate DB and querying the right
+    // DB for the session table exisitence
+    const rows = await this.query(query, [
+      this.options.sessionTableName.toLowerCase(),
+    ]);
+    return Array.isArray(rows) && rows.length === 1;
+  }
+
+  private async migrateSessionTableV1_0_1(): Promise<void> {
```
Comment (cquemin) [inline packages/shopify-app-session-storage-postgresql/src/postgresql.ts:195]: Ideally I merge my [PR](https://github.com/Shopify/shopify-app-js/pull/95) (hint hint for a review) and then you'll have to add this in the migrations file to be run as part of the new migration system :D

**Review by cquemin — COMMENTED** [review]
Small question for me. 
FYI: indirect dependency on https://github.com/Shopify/shopify-app-js/pull/95

**Comment by mkevinosullivan** [comment]
Replaced with #113

**Pull request closed without merging**